src/services/GameEngineManager.ts

```diff
@@ -69,6 +69,28 @@ export const GameEngineManager = {
         };
       }
 
+      return {
+        status: 500,
+        data: undefined,
+      };
+    }
+  },
+  quit: async (lobbyId: string, userId: string): Promise<Response> => {
+    try {
+      const { data } = await gameEngineClient.delete(`/lobbies/${lobbyId}?userId=${userId}`);
```

Comment:
```suggestion
      const { data } = await gameEngineClient.delete(`/lobbies/${lobbyId}`, data);
```

We need to send the user id in the body (like in the join lobby request)

Reply:
I agree